layouts/partials/head.html

```diff
@@ -1,6 +1,7 @@
 <meta http-equiv="Content-Type" content="text/html" charset="UTF-8" />
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
 <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<title>{{ if .Title }}{{ .Title }} | {{ end }}{{ .Site.Title }}</title>
```

Comment:
Can we do `{{ with .Title }}` here?

Also, Maybe make the title something like:

```
{{ .Site.Title }} {{ with .Title }} | {{ . }} {{ end }}
```

That way the tab would be `The Balance | Monk BiS`. Unless we should have the page title first before the `|`?

Reply:
cool cool!